picard/formats/id3.py

```diff
@@ -494,6 +494,19 @@ def _get_tags(self, filename):
         except mutagen.id3.ID3NoHeaderError:
             return compatid3.CompatID3()
 
+    def _load_preserved_config(self, metadata):
+        # Adding additional config to metadata to prevent unecessary saving
+        metadata['~config:rating_steps'] = config.setting['rating_steps']
+        metadata['~config:rating_user_email'] = config.setting['rating_user_email']
```

Comment:
In any case, it occurs to me that this doesn't work anyway. If I change settings, close Picard, open Picard then load files, it will not know that I have changed settings. (So please ignore all my above comments.)

This is NOT the way to check whether e.g. ratings email or images have changed. The way to check this is to store the actual ratings email address in the media file as hidden original metadata and to compare this with the setting at the point of saving. Similarly we need to check images to see if they are the same and saved in the same way at the point of saving.